sdp/processors/toloka/accept_if.py

```diff
@@ -41,24 +42,15 @@ class AcceptIfWERLess(BaseParallelProcessor):
         input_data_file (str): Path to the input data file containing API configurations.
         input_pool_file (str): Path to the input pool file containing pool configurations.
         threshold (float): The WER threshold below which assignments are accepted. Default: 75.
-        config_file (str, optional): Path to the configuration file. Default: None.
-        API_KEY (str, optional): The API key for authenticating with Toloka's API. Default: None.
-        platform (str, optional): The Toloka platform to use. Default: None.
-        pool_id (str, optional): The ID of the Toloka pool. Default: None.
+        config_file (str): Path to the configuration file. Default: None.
+        API_KEY (str): The API key for authenticating with Toloka's API. Default: None.
+        platform (str): The Toloka platform to use. Default: None.
+        pool_id (str): The ID of the Toloka pool. Default: None.
 
     Returns:
         A manifest with accepted assignments from Toloka based on the WER threshold.
-
-    Example:
```

Comment:
This is part of our new documentation format. It look nicely in documentation site
https://nvidia.github.io/NeMo-speech-data-processor/sdp/api.html 
sdp.processors.DuplicateFields[[source]](https://nvidia.github.io/NeMo-speech-data-processor/_modules/sdp/processors/modify_manifest/common.html#DuplicateFields)